test/components/xDragBoxLayerTests.ts

```diff
@@ -1,23 +1,14 @@
 ///<reference path="../testReference.ts" />
 
-describe("Layer Components", () => {
+describe("SelectionBoxLayer", () => {
```

Comment:
Doesn't seem like we have a consistent rule for this. Let's go with "reasonable common ancestor" for now. For example, if you change `SelectionBoxLayer` you'll want to rerun the tests for its subclasses.